source/operatingsystems/linux/mysql/mysql_ssh_tunneling.md

```diff
@@ -0,0 +1,89 @@
+
+## SSH Tunnelling with MySQL Workbench
+
+### Creating a new connection
+- Start by opening MySQL Workbench
+-  Next to MySQL Connection, click the + button. 
+
+Below are the Parameters you will need to fill in:
+
+| Parameters | Description |
+|-|-|
+| Connection Name | A name for the connection that uniquely identifies the connection you're making.  |
+| SSH Hostname | The IP address of the server you're connecting to, followed by the port. With UKFast installations, the default SSH port is 2020. For example, `1.2.3.4:2020` |
+| SSH Username | The username you use to SSH onto the server. |
+| SSH Password/SSH Key File | This is the SSH password or the SSH key for your server. |
+| MySQL Hostname | As we're connecting via an SSH Tunnel, this will likely be 127.0.0.1 |
+| MySQL Server Port | The default for this is 3306 |
+| Username | The MySQL user that you're connecting as. As we're connecting locally, it will need to be a user which has the host set to localhost.  |
+| Password | The MySQL password for the MySQL user.  |
+
+```eval_rst
+.. seealso::
+   Note:
+     When entering passwords, MySQL Workbench offers users the ability to store passwords in a Vault, which is encrypted on your machine. See [MySQL - The Password Storage Vault](https://dev.mysql.com/doc/workbench/en/wb-mysql-connections-vault.html) for more information.
+```
+
+### Testing the connection
+- Once you've entered all the above parameters, click `Test Connection`.
+- You may receive a prompt about the fingerprint not being recognised. If this is the first time connecting to the server and you're confident that the details you've entered are correct, click **Yes**. 
+
+This should return **Successfully made the MySQL connection**. Click **Ok**, and then **Ok** again to save the configuration. If this is unsucessful due to incorrect MySQL credentials, check that the user you're connecting with is set with the host `localhost`.
+
+## SSH Tunnelling with Other Clients (Windows)
+While most MySQL clients support SSH tunnelling, some clients do not. Using PuTTY, we can create an SSH Tunnel systemwide, so that you can connect to MySQL on localhost:3306, which will route the traffic to the server. To do this, follow the following steps: 
+
+- If you don't already have PuTTY, you can download it [here](https://www.chiark.greenend.org.uk/~sgtatham/putty/latest.html). Download the 32/64-bit MSI (most modern operating systems are 64-bit, so if unsure, try this first). Open the downloaded .exe and go through the installation steps.
+ - Once installed, open PuTTY. Below are the values to enter, along with their description. 
+
+ ### Configuring PuTTY with SSH Tunnelling
+
+ #### Session
+ On PuTTY, on the left side, you can see the different categories for values. Start off by selecting Session, and fill in the values below: 
+
```

Comment:
might be worth having a screenshot here of the interface

Reply:
I'll add this in a subsequent PR